.github/workflows/bundleTest-thor.yml

```diff
@@ -78,6 +77,44 @@ jobs:
           sudo chown -R $USER:$USER /opt/HPCCSystems
           sudo xmlstarlet ed -L -u 'Environment/Software/ThorCluster/@slavesPerNode' -v 2 -u 'Environment/Software/ThorCluster/@channelsPerSlave' -v 1 /etc/HPCCSystems/environment.xml
 
+      - name: Checkout ecl-bundles repository
+        uses: actions/checkout@v4
+        with:
+          repository: hpcc-systems/ecl-bundles
+
+      - name: Scrap Bundles List
+        shell: "bash"
+        run: | 
+          TABLES_DATA=$( cat README.rst | grep -oE -e "\|[^\|]*\|"  -e "==.*==" | sed 's/|//g' )
```

Comment:
What happens if no README.rst?

Reply:
I will add condition to check if README.rst file is present. So that it won't fail the workflow file for not having a README.rst file.